packages/core/src/components/notification/notification.css

```diff
@@ -46,6 +48,13 @@
   margin-right: var(--spacing-2-xs);
 }
 
+/* BODY */
+
+.hds-notification__body {
+  font-size: var(--fontsize-body-s);
```

Comment:
This does not match the design in Abstract but then again, neither does the current implementation. I will fix designs to match these.

Reply:
Apparently, this has already been fixed to the implementation - the demo just was not up to date. So disregard this.